rita.rst

```diff
@@ -42,12 +36,12 @@ Run the installer:
 
 ::
 
-   sudo bash ./install.sh
-   
+   sudo bash ./install.sh --disable-bro
```

Comment:
Since Security Onion includes Zeek/Bro already there's no reason to have RITA's installer attempt to install and configure it.